node/src/actors/chain_manager/handlers.rs

```diff
@@ -761,7 +769,7 @@ impl Handler<PeersBeacons> for ChainManager {
                 // we need to rewind one epoch
                 if pb_len == 0 {
                     log::warn!("[CONSENSUS]: We have not received any beacons for this epoch");
-                    self.sm_state = StateMachine::WaitingConsensus;
+                    self.sm_state = StateMachine::AlmostSynced;
                 }
```

Comment:
This `if` may be redundant because in the case with 0 beacons the consensus is `None` and that's handled below.